### ISSUE TITLE
Web app behind password for Grambank coders

### ISSUE THREAD

**Issue by HedvigS** [HedvigS/gramfinder-typology-of-terminnology-and-searching-OCRed-grammars#6]
One of the concrete plans of the meeting me, @skalyan91 and @tyrannomark  had was that we should have a simple web app only for Grambankers that searchers multiple resources for a language. Presumably, it needs to be at the Grambank-website but behind password, was that what you had in mind Siva?

Currently, @xrotwang and @d97hah have created some new tools on the Grambank website for coders, and they're behind passwords because the entire site is, correct? Would we need some sort of separate "login" on the site for later when it becomes public?

**Comment by skalyan91** [comment]
Yes, that's what I had in mind. I take it this will be the first stage of "Gramfinder".

Gramfinder as a whole will probably need to be behind a password even after Grambank goes public.

**Comment by HedvigS** [comment]
@skalyan91 judging by recent emails with @xrotwang am I right to assumed we should put this on the backburner until search scripts are more refined, and implement a web app later?

**Comment by skalyan91** [comment]
Yup.

**Comment by d97hah** [comment]
I don't think that's a necessary conclusion. What Robert said is that it's
a too big web app with individual user accounts, input and output
capabilities etc. A CLLD app with just one password and no user editing
facilities, however, is a much leaner task. You can develop it on your own
computer (there's even a step-by-step how-to) and then when all is done
push it live via Robert but with little effort required on his part (though
he does like the code to be neat & clean and lint-free). Everyone has
different styles but I like to develop front and back end at the same time,
esp when it's not written in stone what the full functionality will be at
the front end. all the best, H
